src/tickit_devices/eiger/stream/eiger_stream.py

```diff
@@ -28,54 +43,110 @@ class EigerStream:
 
     def __init__(self, callback_period: int = int(1e9)) -> None:
         """An Eiger Stream constructor."""
-        self.stream_status = StreamStatus()
-        self.stream_config = StreamConfig()
-        self.stream_callback_period = SimTime(callback_period)
-
+        self.status = StreamStatus()
+        self.config = StreamConfig()
+        self.callback_period = SimTime(callback_period)
 
-class EigerStreamAdapter:
-    """An adapter for the Stream."""
+        self._message_buffer = Queue()
 
-    device: EigerStream
-
-    @HTTPEndpoint.get(f"/{STREAM_API}" + "/status/{param}")
-    async def get_stream_status(self, request: web.Request) -> web.Response:
```

Comment:
Are we no longer exposing this endpoint?

Reply:
It was moved into `eiger_adapters.py`. I replaced multiple inheritance with one big class.